### PR TITLE
uploading project

### PR DESCRIPTION
**Pull request by MajdHassoun**

### CODE REVIEW

File: library task/book.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Book:
+
```
Comment (ShibelAl) [inline library task/book.py:2]: nice space.

File: library task/library.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 from book import Book
 
-
 class Library:
```
Comment (ShibelAl) [inline library task/library.py]: good job!

**Review by ShibelAl — COMMENTED** [review]
nice code majd!

File: library task/library.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
 from book import Book
 
-
```
Comment (xMeliiodaS) [inline library task/library.py]: Should add more line between the import and the class

File: library task/app.py [hunk 4]
```diff
@@ -6,7 +6,6 @@
 app = Flask(__name__)
 library = load_library()
 
-
```
Comment (xMeliiodaS) [inline library task/app.py]: Please remove this space, its uncessary

File: library task/storage.py [hunk 5]
```diff
@@ -3,7 +3,6 @@
 
 FILE_PATH = 'library.json'
 
-
```
Comment (xMeliiodaS) [inline library task/storage.py:6]: Not cool :(
Please fix before the day end or you are fired.
If you do, you are watered !!!

**Review by xMeliiodaS — CHANGES_REQUESTED** [review]

**Review by xMeliiodaS — APPROVED** [review]
Good job !!

File: library task/storage.py (hunk 5 above)
Comment (ShibelAl) [reply library task/storage.py:6]: You are fired

**Pull request merged**